### ISSUE TITLE
Fix S6588 FN: Rule should cover case with epoch ticks

### ISSUE THREAD

**Issue by csaba-sagi-sonarsource** [SonarSource/sonar-dotnet#7547]
### Description

The rule should raise if the constructor is called with the ticks that are equal to the epoch ticks (621355968000000000).

```cs
const long EpochTicks = 621355968000000000;

_ = new DateTime(621355968000000000); // Noncompliant
_ = new DateTime(EpochTicks); // Noncompliant
```

While doing the validation for rule S6562 I found examples like: _ = new DateTime(EPOCH);
